MIGRATING.md

```diff
@@ -0,0 +1,214 @@
+# Migration Guide
+
+When we update `@fastly/compute-js-static-publish` to a new major version, sometimes we make changes to the files that
+are generated during scaffolding. For this reason, if possible we recommended that you re-scaffold your application.
+
+This is pretty simple if you've been using your `compute-js-static-publisher` application without modification, but we
+know that's not always possible, so the following is provided to help you overcome any breaking changes.
```

Comment:
```suggestion
New major versions of `@fastly/compute-js-static-publish` can involve changes to the files that
are generated during scaffolding. For this reason, it is recommended that you re-scaffold your application.

This is straightforward if you're using `compute-js-static-publisher` out-of-the-box. Otherwise, read on.
```